package.json

```diff
@@ -32,6 +32,14 @@
     "test-multibrowser": "grunt test:multibrowser",
     "test-snyk": "snyk test"
   },
+  "files": [
+    "bin",
+    "build",
+    "lib",
+    "index.js",
```

Comment:
including `lib` is nice because I beleive that ES6-code can be natively in future.